latest/index.bs

```diff
@@ -24,19 +24,13 @@ Status Text: will be provided between numbered versions. Data written with these
 Status Text: (an "editor's draft") will not necessarily be supported.
```

Comment:
Row above: `Status Text: <a href="../0.4/index.html">0.4</a>.` looks like it needs manual update to `0.5`?

Same for line 612: `This edition of the specification is [https://ngff.openmicroscopy.org/0.4/](https://ngff.openmicroscopy.org/0.4/]).`